### PR TITLE
test: test get current language from url

### PR DESCRIPTION
**Pull request by gabaldon**

### CODE REVIEW

File: tests/utils/components/LanguageSelector/getCurrentLanguage.spec.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    const expectedLanguage = {
+      code: 'es',
+      iso: 'es-ES',
+      label: 'Español',
+      name: 'Español',
+    }
+    expect(getCurrentLanguage(url)).toStrictEqual(expectedLanguage)
+  })
+  it('should return the correct language', () => {
```
Comment (Tommytrg) [inline tests/utils/components/LanguageSelector/getCurrentLanguage.spec.js:15]:
```suggestion
  it('should return the correct language when no locale is specified in the url', () => {
```

File: tests/utils/components/LanguageSelector/getCurrentLanguage.spec.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { getCurrentLanguage } from '../../../../utils'
+
+describe('getCurrentLanguage', () => {
+  it('should return the correct language', () => {
```
Comment (Tommytrg) [inline tests/utils/components/LanguageSelector/getCurrentLanguage.spec.js:4]:
```suggestion
  it('should return the correct language when the locale is in the url', () => {
```

File: utils.js [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
-  return languages.find((language) => language.code === locale)
+import { defaultLocale } from './default'
+import { languages } from './constants'
+
+export function findLanguage(locale) {
+  return getExpandedLanguages().find((language) => language.code === locale)
+}
+
+export function getLanguageFromUrl(url) {
+  return url.split('/')[1] || defaultLocale
```
Comment (Tommytrg) [inline utils.js:9]:
```suggestion
  return url.split('/')[1]
```

Comment (gabaldon) [reply utils.js:9]: if no locale is specified in the url it should return the default

**Pull request merged**